src/Vertex.js

```diff
@@ -144,15 +174,56 @@ class Vertex extends Component {
         !!sign && markerMarkup()
       ),
 
-      !!paint &&
+    !!paint &&
+    h('div', {},

```

Comment:
I really like this enhancement, however we're adding a lot of new DOM elements just for a single painted vertex, which I don't like. It would be cool if you could find a way use CSS pseudo-elements instead...